clientUtils/Util.ts

```diff
@@ -1037,3 +1037,15 @@ export const differenceObj = <
     }
     return result
 }
+
+export const getDOMParent = (
```

Comment:
I would name this `findDOMParent`, because there is a condition passed (mirroring [`Array.prototype.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)). 

`getDOMParent` seems like something that would just go one level up.

Reply:
makes sense, thanks! Will do.